sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -107,6 +107,10 @@ export interface OnError {
  * @class MessageReceiver
  */
 export class MessageReceiver extends LinkEntity {
+  /**
+   * @property {boolean} isClosed Returns true if this receiver is closed.
+   */
+  isClosed: boolean;
```

Comment:
Do we need this variable anymore?
It is currently used only in `receive()` in the Streaming Receiver which is called from `registerMessageHandler` which is making a similar check

Reply:
Yes, this is now unnecessary with the init() move.